backend/lib/endpoints/posts.js

```diff
@@ -113,15 +128,30 @@ async function routes(app) {
       }
 
       //workMode filter
-      if (workMode){
-        const workModes = workMode.map(mode => mode.toLowerCase())
-        if(workModes.includes('both')){
-          filters.push({ $or: [ { $and: [ { workMode: null }, {types: { $nin : ['Remote Work'] } } ] }, {workMode: { $in: workModes } }] });
-        }
-        else if(workModes.includes('remote')) {
-          filters.push({ $or: [ { $and: [ { workMode: null }, {types: { $in : ['Remote Work'] } } ] }, {workMode: { $in: workModes } }] });
-        }
-        else {
+      if (workMode) {
+        const workModes = workMode.map((mode) => mode.toLowerCase());
+        if (workModes.includes("both")) {
+          filters.push({
+            $or: [
+              {
+                $and: [
+                  { workMode: null },
+                  { types: { $nin: ["Remote Work"] } },
+                ],
+              },
+              { workMode: { $in: workModes } },
+            ],
+          });
+        } else if (workModes.includes("remote")) {
+          filters.push({
```

Comment:
this `else if` filter is different from above one only at the $nin and $in part, maybe use a conditional statement to simplify it

Reply:
Done.